src/components/Icon/selection.json

```diff
@@ -0,0 +1,183 @@
+{
+    "IcoMoonType": "selection",
```

Comment:
I hope this is just a sample selection.json.

Reply:
Yes it is, the actual icon set needs to be passed through config provider